packages/components/tests/dummy/app/templates/components/form/text-input.hbs

```diff
@@ -0,0 +1,462 @@
+              />
+              <Hds::Form::TextInput::Base
+                placeholder="Placeholder"
+                disabled={{if (eq variant "disabled") "disabled"}}
+                readonly={{if (eq variant "readonly") "readonly"}}
+                @isInvalid={{if (eq variant "invalid") true}}
+              />
+              <Hds::Form::TextInput::Base
+                @value="Lorem ipsum dolor"
+                disabled={{if (eq variant "disabled") "disabled"}}
+                readonly={{if (eq variant "readonly") "readonly"}}
+                @isInvalid={{if (eq variant "invalid") true}}
+              />
+              <Hds::Form::TextInput::Base
+                @type="date"
+                @value="Lorem ipsum dolor"
+                disabled={{if (eq variant "disabled") "disabled"}}
+                readonly={{if (eq variant "readonly") "readonly"}}
+                @isInvalid={{if (eq variant "invalid") true}}
+              />
+              <Hds::Form::TextInput::Base
+                @type="time"
+                @value="Lorem ipsum dolor"
+                disabled={{if (eq variant "disabled") "disabled"}}
+                readonly={{if (eq variant "readonly") "readonly"}}
+                @isInvalid={{if (eq variant "invalid") true}}
+              />
+            </div>
+          </div>
+        {{/each}}
+      {{/each}}
+    {{/let}}
+  </div>
+  <h5 class="dummy-h6">Custom layout</h5>
+  <div class="dummy-form-text-input-base-sample">
+    <div>
+      <span class="dummy-text-small">With custom layout</span>
+      <br />
+      <div class="dummy-form-text-input-custom-layout">
+        <label for="my-custom-text-input-example">Custom label</label>
+        <Hds::Form::TextInput::Base id="my-custom-text-input-example" @value="Lorem ipsum dolor" />
+        <span class="dummy-form-text-input-custom-layout__append-text">Some content</span>
+      </div>
+    </div>
+  </div>
+  <h5 class="dummy-h5">Containers</h5>
+  <div class="dummy-form-text-input-containers">
+    {{#let (array "block" "flex" "grid") as |displays|}}
+      {{#each displays as |display|}}
+        <div>
+          <span class="dummy-text-small">Parent with <code class="dummy-code">display: {{display}}</code></span>
+          <br />
+          <div class="dummy-form-text-input-sublist">
+            <div class="dummy-form-text-input-containers__{{display}}">
+              <Hds::Form::TextInput::Base @value="Default width" />
+            </div>
+            <div class="dummy-form-text-input-containers__{{display}}">
+              <Hds::Form::TextInput::Base @value="Custom width" @width="248px" />
+            </div>
+            <div class="dummy-form-text-input-containers__{{display}}">
+              <Hds::Form::TextInput::Base @type="date" />
+            </div>
+            <div class="dummy-form-text-input-containers__{{display}}">
+              <Hds::Form::TextInput::Base @type="time" />
```

Comment:
```suggestion
              <Hds::Form::TextInput::Base @type="time" aria-label="{{display}}" />
```

Reply:
As discussed during the code review today, I am not convinced this is the way to go for the "showcase" components. I totally understand the idea to have accessible documentation pages, but the _show_case is intended mainly to _show_ the components, with a particular focus on the developers implementing the components (and Percy) plus as a bonus side-effect, show to the consumers what are the variants (and their combinations) of a component. 

The issue of devs copying&pasting the code from the showcase is minimal, because we don't expect them to copy the HTML, and (unless I'm missing something) is not possible to copy the code from the Ember inspector.

So the last problems remains having pages which will not pass an accessibility test. Here I can propose some alternatives:
- we hide the whole showcase from the accessibility tree (a bit drastic!)
- we hide the parent container/wrapper of the "base" components from the accessibility tree (more specific)
- we accept that not all the "base" elements in the showcase will meet all the accessibility criteria, when considered in isolation
  - this is not something unusual: consider this MDN page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input - as you can see the top example has a label associated with the `<input>` but the ones in the list of types they don't and they also don't have a `aria-label` associated, and I am not sure I would expect them to have it frankly (their purpose meant to showcase the component, not as real usable/interactive content).